docs/team/jonahtanjz.md

```diff
@@ -1,46 +1,69 @@
-  * Justification: This feature improves the product significantly because a user can make mistakes in commands and the app should provide a convenient way to rectify them.
-  * Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
-  * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+## Overview
 
-* **New Feature**: Added a history command that allows the user to navigate to previous commands using up/down keys.
+SimplyKitchen is a desktop app for food inventory management, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). With intuitive and practical features, SimplyKitchen can get food management tasks done faster and more efficiently than traditional GUI apps.
+Users interact with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
 
-* **Code contributed**: [RepoSense link]()
+This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org/).
 
-* **Project management**:
-  * Managed releases `v1.3` - `v1.5rc` (3 releases) on GitHub
+## Summary of contributions
 
-* **Enhancements to existing features**:
-  * Updated the GUI color scheme (Pull requests [\#33](), [\#34]())
-  * Wrote additional tests for existing features to increase coverage from 88% to 92% (Pull requests [\#36](), [\#38]())
+### New Features:
 
-* **Documentation**:
+  * Added **Priority attribute to food items**.
+    * What it does: Allows the user to specify a priority, either high, medium or low, for a food item.
+    * Justification: This feature provides a convenient way for the user to mark the importance of different food items.
+    * Highlights: This enhancement affects existing commands and commands to be added in future. It required the parser to identify a new prefix and perform necessary validation to ensure that the priority specified in the command is valid.
+
+  * Added a **popup window to show a list of expired food items**.
+    * What it does: If there are expired items in the inventory, a popup will automatically be launched during the start-up of the application, showing a list of expired food items. This popup can also be launch using the 'expired' command.
+    * Justification: This feature alerts the user of expired items in their inventory whenever the application is launched so that they will be reminded to dispose these items and not accidentally consume them. The user can also open this popup using the 'expired' command so that they can access this list at any point of time.
```

Comment:
Perhaps use `expired` to standardise with line 54

Reply:
Ok will change.